content/metadata.md

```diff
@@ -79,12 +81,35 @@ An example of how to identify the cover image is as follows:
 * __accessibilityFeature__: Content features of the resource, such as accessible media, alternatives and supported enhancements for accessibility. (WebSchemas wiki lists possible values). (schema.org/Text)
 * __accessibilityHazard__: A characteristic of the described resource that is physiologically dangerous to some users. Related to WCAG 2.0 guideline 2.3. (WebSchemas wiki lists possible values). (schema.org/Text)
 
-Example of a metadata csv-file:
+#### Example:
 ```csv
 Title, Link to epub, Reading Level
 "676719b9-f5bb-4ce7-83ca-e221cbc53c67", "Bath time for Chunnu and Munnu", https://books.digitallibrary.io/epub/en/676719b9-f5bb-4ce7-83ca-e221cbc53c67.epub, 1
 "c418ac9f-fa75-4a72-9f3a-8ceacee850f2", "A Street or a Zoo?", https://books.digitallibrary.io/epub/en/c418ac9f-fa75-4a72-9f3a-8ceacee850f2.epub, 1
 ```
+
+## Games
+The GDL can host information about games and how to install/play the games.
```

Comment:
Foreslår følgende: The GDL platform includes a separate category for games in selected languages, including information about the games and how to install/play them.